### ISSUE TITLE
Fix datetime usage for locale

### ISSUE THREAD

**Issue by dgets** [dgets/lasttime#103]
Now that we're working with per-diem constraints on the graphing information, it appears that our use of non-localized `datetime` objects is going to be causing some problems in _dataview/dump_constrained_dose_graph_data_.  We could code manually around this, but that'd be locked to this timezone and would be rather disgusting.  It's a better solution to go and set the timezone correctly; this will probably entail asking the user at initial account creation what their locality is.

This is currently messing up any administration per-diem totals that include administrations past a certain time late in the day while working on #97.

**Comment by dgets** [comment]
See also: [py3 datetime object API](https://docs.python.org/3/library/datetime.html), which will be necessary for reference, as this is going to be the first time working with a localized `datetime` object in python for me.

**Comment by dgets** [comment]
Despite the blocking mentioned in the last commit, I believe that I'm just going to set a static timezone for now, so that I can return to the thread that I was working on.  After completion of that (issue #97), I'll fully implement the timezone awareness.

**Comment by dgets** [comment]
I started trying to implement a bit for the _create_user_ view to be able to allow the user to select a timezone.  Right now it's just a stub, being as the actual timezone data is set from `myglobals.Const.Time_Zone`, though.  I did try to test it out and couldn't find any place in the user records that it seemed to have saved that field, however; I'm not sure if I need to define a custom user model or what, but I think something else needs to be done here as well.

Regardless, the constant is working enough for me to get back working on #97, so I'll start back up on this afterwards.
